### DISCUSSION TITLE
60 Hz pickup

### DISCUSSION THREAD

**Discussion by eehealy** [simonsobs/sodetlib#359] [Issues and Debugging]
After soft hammering, I am seeing pickup at 60 Hz that wasn't there before hammering. Nothing in the setup was touched except via smurf. Does this indicate a "real" short that just appeared or could this be something going on with smurf?

noise before hammering:
![image](https://github.com/simonsobs/sodetlib/assets/33061247/d046f98c-333f-4216-9d98-91be4e1d1bc1)

noise after hammering:
![image](https://github.com/simonsobs/sodetlib/assets/33061247/2fe298fa-0e61-42f3-b6a8-9142549c1d89)

**Comment by msilvafe** [comment]
> After soft hammering, I am seeing pickup at 60 Hz that wasn't there before hammering. Nothing in the setup was touched except via smurf. Does this indicate a "real" short that just appeared or could this be something going on with smurf?
> 
> 
> 
> noise before hammering:
> 
> ![image](https://github.com/simonsobs/sodetlib/assets/33061247/d046f98c-333f-4216-9d98-91be4e1d1bc1)
> 
> 
> 
> noise after hammering:
> 
> ![image](https://github.com/simonsobs/sodetlib/assets/33061247/2fe298fa-0e61-42f3-b6a8-9142549c1d89)
> 
> 

Interesting...when soft hammering a number of things aren't configured that are when you run S.setup(). Did you run that after your hammer? If not can you try then tune up again and compare noise? If that fixes it will have to do some thinking about what in the Smurf settings side would cause this

**Reply by eehealy** [reply]
Do you mean the uxm_setup function or something else? 

I just tried running uxm_setup twice, once failed for "Exception: Timeout waiting for PVs to update". Second time failed with "ValueError: Playing the noise file saturates the ADC for band 0.  Try increasing the DC attenuation for this band."

If I have to set a new DC attenuation, could you provide instructions / point to functions on how to do that? Do you know what would cause the first failure?

**Reply by eehealy** [reply]
Hmm... something changed my DC attenuation on band 0 to zero. I ran S.get_att_dc(0) before and after running setup and it went from 22 to 0.

**Comment by jlashner** [comment]
I am skeptical that something in the smurf settings can cause this spike... When were these two noise data sets taken? If the initial measurement without the 60 Hz signal was from before the LN2 tests, then I think it's very possible that this signal existed before the soft-hammer, but we didn't see it until we were able to fix the noise issues.

**Reply by msilvafe** [reply]
I would second that I’m skeptical that it’s from Smurf things but was just trying to take your statement “nothing in the setup was touched” very literally

**Reply by eehealy** [reply]
The first plot above is after running relock after the LN2 test. 

Here's is the LN2 noise again. I still think the noise is low enough out at 60 Hz that we can see it's not buried. 
![image](https://github.com/simonsobs/sodetlib/assets/33061247/20bfe156-0d29-4e02-8474-a8e4a8a4d681)

**Comment by jlashner** [comment]
Hmmm yeah I think you're right, we probably should've seen this even with the high noise. I dont think it's an incorrect setting, but hammering does tend to rail the bias voltages, so it's possible that induced a short. I think the best way to check if it's real is to probe the bias lines the next time you have the chance... I think the bias voltage railing is currently being fixed in pysmurf, so we should update our version as soon as that's done. I think you should be able to continue with detector operation even with the ground short for the time being

**Comment by msilvafe** [comment]
Did this ever get resolved @eehealy ?
